core/controllers/topic_viewer.py

```diff
@@ -26,20 +26,16 @@ class TopicViewerPage(base.BaseHandler):
     """Renders the topic viewer page."""
 
     @acl_decorators.can_access_topic_viewer_page
-    def get(self, topic_name):
+    def get(self, _):
         """Handles GET requests."""
 
         if not constants.ENABLE_NEW_STRUCTURE_PLAYERS:
             raise self.PageNotFoundException
 
-        topic = topic_services.get_topic_by_name(topic_name)
-
-        self.values.update({
-            'topic_name': topic.name
-        })
         self.render_template('dist/topic_viewer.html')
 
 
+
```

Comment:
This is accidental? Remove the line please.

Reply:
Yes, okay